Tasks/VSMobileCenterUpload/vsmobilecenterupload.ts

```diff
@@ -6,7 +6,7 @@ import fs = require('fs');
 
 import { ToolRunner } from 'vsts-task-lib/toolrunner';
 
-var utils = require('./utils.js');
+import utils = require('./utils');
```

Comment:
I had some problem mocking this if it's "import"ed.  You can try and see if mock works.